pubspec.yaml

```diff
@@ -31,3 +31,7 @@ dev_dependencies:
 executables:
   dartfmt: format
   dartformat: format # Allow the old name for compatibility.
+
+# For CI - remove before commiting!
+dependency_overrides:
+  analyzer: ^2.0.0
```

Comment:
What's this about?

Reply:
Forcing CI to run w/ v2.0.0 (it's blocked by pkg:test at the moment) to make sure there are no breaks.

I can nix this now, or you can merge this into your PR. Let CI run w/ it, then remove it.

Just to make sure we have coverage of the new API

Reply:
SGTM. I'll merge this into the "bump" branch. Once CI looks good there, I'll remove this and then merge to master and publish.